_layouts/base.html.haml

```diff
@@ -30,6 +30,17 @@
     %link{:rel => "shortcut icon", :href => relative("/favicon.ico")}
 
   %body(data-spy="scroll" data-target=".bs-docs-sidebar")
+<!--Place GTM Code just below the opening body tag -->
+<script>dataLayer = [{'channel' : 'OptaPlanner', 'additional_tracking_code' : '#{site.googleAnalyticsId}'}];</script>
```

Comment:
Moving the dataLayer line inside the other script to conserve bandwidth (but still before the GTM script as required by the GTM docs). It doesn't need to be before the noscript element as that only executes if javascript is disabled.